smartseq2_single_sample/build_references/build_hisat2_reference/BuilHISAT2TranReferencebundle.wdl

```diff
@@ -0,0 +1,45 @@
+task BuildHISAT2Trans {
+  String ref_name ## name of the tar.gz file without tar.gz suffix
```

Comment:
How would you feel about standardizing this so that it's determined by the genome's `gtf_version` instead of being passed as a separate parameter?

Reply:
possible.

Reply:
try to keep more options, especially if we deal with multiple genome+annotation+species, such as human or mouse.

Reply:
Ok, that makes sense. let's leave as is.